mbed_host_tests/host_tests_plugins/host_test_plugins.py

```diff
@@ -151,7 +151,7 @@ def check_mount_point_ready(self, destination_disk, init_delay=0.2, loop_delay=0
                 self.print_plugin_info("Mount point for '%s' changed from '%s' to '%s'..."% (target_id, destination_disk, new_destination_disk))
                 destination_disk = new_destination_disk
 
-        result = False
+        result = True
         # Check if mount point we've promoted to be valid one (by optional target_id check above)
         # Let's wait for 30 * loop_delay + init_delay max
         if not access(destination_disk, F_OK):
```

Comment:
I guess the whole point of that `if` statement is to prevent the printing of "Waiting for mount point..." if its not necessary.  Do you still want to make this change @mazimkhan?